_data/downloads.yml

```
@@ -1,29 +1,21 @@
 stepOne:
   - os: linux
-    text: "<p><code>java -version</code><span>(Make sure you have version 1.8.)</span></p>"
+    text: >-
+      <code class="hljs">$ curl -Lo cs https://git.io/coursier-cli-linux && chmod +x cs && ./cs setup </code> <br>
   - os: unix
-    text: "<p><code>java -version</code><span>(Make sure you have version 1.8.)</span></p>"
+    text: >-
+            <p>Follow <a href="https://get-coursier.io/docs/cli-overview.html#install-native-launcher" target="_blank">the instructions to install the <code>cs</code> launcher</a> then run:</p>
+            <p><code>$ ./cs setup</code></p>
   - os: macos
-    text: "<p><code>java -version</code><span>(Make sure you have version 1.8.)</span></p>"
+    text: >-
+      <div class="highlight">
+        <code class="hljs">$ brew install coursier/formulas/coursier && cs setup </code> <br>
+        </div>
+        <p>Alternatively, if you don't use Homebrew</p>
+        <div class="highlight">
+        <code class="hljs">$ curl -Lo cs https://git.io/coursier-cli-macos && chmod +x cs &&  (xattr -d com.apple.quarantine cs || true) && ./cs  setup </code> <br>
+      </div>
```

The indentation is misleading.
```suggestion
        <code class="hljs">$ brew install coursier/formulas/coursier && cs setup </code> <br>
      </div>
      <p>Alternatively, if you don't use Homebrew</p>
      <div class="highlight">
        <code class="hljs">$ curl -Lo cs https://git.io/coursier-cli-macos && chmod +x cs &&  (xattr -d com.apple.quarantine cs || true) && ./cs  setup </code> <br>
      </div>
```

---

ok. thanks